server/hydrate/generator.js

```diff
@@ -43,6 +43,10 @@ const userNameFrags = [
 'clothes',
 'shits',
 'lol',
+<<<<<<< HEAD
+'butts',
```

Comment:
have to finish resolving this merge conflict. Some of the merge conflict code is still in this file